quay-enterprise/quay-ssl.md

```diff
@@ -0,0 +1,136 @@
+# SSL + Quay Enterprise
+
+This document assumes you have deployed [Quay Enterprise as a single container.](https://tectonic.com/quay-enterprise/docs/latest/initial-setup.html)
```

Comment:
full stops go outside of the links , e.g., `[Quay Enterprise as a single container](linktarget).

Preferably, write reference-style markdown links, see: https://github.com/coreos/docs/blob/master/STYLE.md#marking-down-the-link